lib/peripheral.dart

```diff
@@ -4,19 +4,38 @@ abstract class _PeripheralMetadata {
   static const name = "name";
   static const identifier = "id";
 }
-
+/// Device instance represents BLE peripheral
 class Peripheral {
   static const int NO_MTU_NEGOTIATION = 0;
   ManagerForPeripheral _manager;
 
   String name;
   String identifier;
 
+  /// Creates Peripheral from JSON
+  ///
+  /// Creates peripheral from [json] and initializes it with [manager]
   Peripheral.fromJson(Map<String, dynamic> json, ManagerForPeripheral manager)
       : _manager = manager,
         name = json[_PeripheralMetadata.name],
         identifier = json[_PeripheralMetadata.identifier];
 
+  /// Connects to the Device.
+  ///
+  /// Optional [isAutoConnect] controls whether to directly connect to the
+  /// remote device (false) or to automatically connect as soon as the
+  /// remote device becomes available (true). [Android only]
+  /// Optional [requestMtu] size will be negotiated to this value. It is not
+  /// guaranteed to get it after connection is successful. 0 passed as
+  /// Optional [requestMtu] means that the MTU will not be negotiated
+  /// Passing true as [refreshGatt] leads reset services cache. This option may
+  /// be useful when a peripheral's firmware was updated and it's
+  /// services/characteristics were added/removed/altered. [Android only]
```

Comment:
```suggestion
  /// services/characteristics were added/removed/altered (Android only).
```
As `[...]` formats text as code we should not use it for regular text.